features/patroni_api.feature

```diff
@@ -35,21 +35,23 @@ Scenario: check local configuration reload
 	Then I receive a response code 202
 
 Scenario: check dynamic configuration change via DCS
-	Given I run patronictl.py edit-config -s 'ttl=10' -p 'max_connections=101' --force batman
-	Then I receive a response returncode 0
-	And I receive a response output "+ttl: 10"
+	Given I issue a PATCH request to http://127.0.0.1:8008/config with {"ttl": 20, "postgresql": {"parameters": {"max_connections": "101"}}}
+	Then I receive a response code 200
+	And I receive a response output "ttl: 20"
+	And I receive a response output "max_connections: 101"
```

Comment:
@feikesteenbergen behave tests are failing because of this.
In my suggestion there was no these lines